docs/src/main/tut/testing.md

```diff
@@ -0,0 +1,89 @@
+---
+menu: main
+weight: 350
+title: Testing
+---
+
+## Introduction
+
+Testing http4s [[org.http4s.HttpService]]'s is straightforward given that it's:
+
+```scala
+    type HttpService[F[_]] = Kleisli[OptionT[F, ?], Request[F], Response[F]]
```

Comment:
This may be getting tweaked soon (see discussion on #1692), but is true for 0.18.x.  This PR might be better targeted on top of that branch.

Reply:
Thanks - will make sure I open the PR off the right branch!